cadc-vos-server/src/main/java/org/opencadc/vospace/server/actions/NodeAction.java

```diff
@@ -131,30 +137,10 @@ public void initAction() throws Exception {
     }
     
     @Override
-    protected final InlineContentHandler getInlineContentHandler() {
+    protected InlineContentHandler getInlineContentHandler() {
         return new InlineNodeHandler(INLINE_CONTENT_TAG);
     }
-    
-    // the absolute URI constructed from the request path
-    protected final VOSURI getTargetURI() {
```

Comment:
why move this to Utils and make all thiose other little changes? it doesn't seem related

Reply:
Reverted. Found a replacement of this method for `vault`